src/commands/rsync.ts

```diff
@@ -70,15 +70,17 @@ export const rsyncCommand = async () => {
 		return;
 	}
 
-    const confirmation = await vscode.window.showWarningMessage(
-        `You're about to upload the contents of plugins/${plugin} to ${wpPath}. Do you want to proceed?`,
-        'Yes',
-        'No'
-    );
-
-    if (confirmation !== 'Yes') {
-        return;
-    }
+	if (existingSite !== newSiteOption) {
```

Comment:
```suggestion
	if (existingSite == newSiteOption) {
```
I think this does the opposite of what we want right? If we are selecting newSiteOption, we want to show the notice?